docs/Documentation/06-publishing.md

```diff
@@ -1,15 +1,9 @@
 # Publishing in Studio
 
-<!-- theme: danger -->
-
-> Files **must** be stored in a Git repository to be published
+Instantly Publish API Descriptions, Models, and supplemental Markdown files and images. One-click publish significantly reduces time to production and produces beautiful, modern, developer-centric documentation.
```

Comment:
We actually don't publish your images yet. We currently try to load them directly from your VCS which doesn't work for private projects. We've got plans to support uploading images to our CDN though!

```suggestion
Instantly Publish API Descriptions, Models, and supplemental Markdown files. One-click publish significantly reduces time to production and produces beautiful, modern, developer-centric documentation.
```